### ISSUE TITLE
Not found: FeedInputs: unable to find feed output Mul

### ISSUE THREAD

**Issue by amirjamez** [tensorflow/tensorflow#15651]
@satok16 I have already set up and was able to run `hexagon_graph_execution` on my hvx board, however, when I tried to use my own [inception-v3 pre-trained model](https://github.com/tensorflow/models/tree/master/research/slim#pre-trained-models) that I froze using [this quantization method](https://www.tensorflow.org/performance/quantization), I am receiving this error:

```
[ RUN      ] GraphTransferer.RunInceptionV3OnHexagonExampleWithTfRuntime
native : hexagon_graph_execution_test.cc:519 Fuse and run inception v3 on hexagon with tf runtime
native : hexagon_graph_execution_test.cc:94 Hexagon controller version is 90
native : hexagon_graph_execution_test.cc:142 Read /data/local/tmp/img_299x299.bmp, size = 269156bytes
native : hexagon_graph_execution_test.cc:148 header size = 54
native : hexagon_graph_execution_test.cc:150 image size = 40
native : hexagon_graph_execution_test.cc:152 width = 299
native : hexagon_graph_execution_test.cc:154 height = -299
native : hexagon_graph_execution_test.cc:533 Ioading image finished.
t1(loading image time)=0.026770
native : hexagon_graph_execution_test.cc:546 Build fused graph
native : remote_fused_graph_execute_utils.cc:259 Error during inference: Not found: FeedInputs: unable to find feed output Mul
native : graph_transfer_utils.cc:110 Check failed: status.ok()
Aborted
```
Do you know if the issue is because of an incorrect input argument here: 

```
curl -L "https://storage.googleapis.com/download.tensorflow.org/models/inception_v3_2016_08_28_frozen.pb.tar.gz" |
  tar -C tensorflow/examples/label_image/data -xz
bazel build tensorflow/tools/graph_transforms:transform_graph
bazel-bin/tensorflow/tools/graph_transforms/transform_graph \
  --in_graph=tensorflow/examples/label_image/data/inception_v3_2016_08_28_frozen.pb \
  --out_graph=/tmp/quantized_graph.pb \
  **--inputs=input \**
  --outputs=InceptionV3/Predictions/Reshape_1 \
  --transforms='add_default_attributes strip_unused_nodes(type=float, shape="1,299,299,3")
    remove_nodes(op=Identity, op=CheckNumerics) fold_constants(ignore_errors=true)
    fold_batch_norms fold_old_batch_norms quantize_weights quantize_nodes
    strip_unused_nodes sort_by_execution_order'
```

I found on [this] (https://stackoverflow.com/questions/43022516/tensorflow-inception-feedinputs-unable-to-find-feed-output-input) and also [this][https://github.com/tensorflow/tensorflow/issues/2883#issuecomment-226591095](url) posts that we might have to use `Mul` instead. Tried that with no success. Interestingly, when I test my frozen_quantized graph with:

`bazel-bin/tensorflow/examples/label_image/label_image --graph=/tmp/my_inception_quantized_graph_hvx.pb`

I receive similar results compared to a non-quantized version, so it shows that my frozen_quantized is not faulty. Can you verify the issue here? 
Was the file `https://storage.googleapis.com/download.tensorflow.org/models/tensorflow_inception_v3_stripped_optimized_quantized.pb` used in the original hvx hexgon_graph_execution produced differently?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activityand the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @skye: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by amirjamez** [comment]
I am trying to build the more recent `libhexagon_nn_skel.so` version mentioned in [TF-HVX](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/hvx/), however, I am having a compilation error: 

`make[1]: Entering directory `/home/aashouri/Qualcomm/Hexagon_SDK/3.0/examples/common/nnlib'
ERROR: Could not open file (/prj/dsp/qdsp6/arch/cnn/setup/inceptionv3_uint8in.c)
make[1]: *** [android_Release/inceptionv3_uint8in.o] Error 255`

working around by defining a dummy input data as mentioned by @satok16 [here](https://github.com/tensorflow/tensorflow/issues/9428) did not solve my issue neither.

**Comment by skye** [comment]
@suharshs are you able to comment on this? Thanks.

**Comment by suharshs** [comment]
We have focused quantization efforts to https://www.tensorflow.org/performance/quantization with support in [TensorFlow Lite](https://www.tensorflow.org/mobile/tflite/).

That being said, your current issue seems specific to contrib/hvx. @satok16 @petewarden @andrewharp do you have any ideas what could be the issue here?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 31 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 46 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 61 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 76 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 91 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 106 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @petewarden, @suharshs, @rockyrhodes: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @petewarden, @suharshs, @rockyrhodes: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @petewarden, @suharshs, @rockyrhodes: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @petewarden, @suharshs, @rockyrhodes: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @petewarden, @suharshs, @rockyrhodes: It has been 74 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @petewarden, @suharshs, @rockyrhodes: It has been 91 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignees @petewarden, @suharshs, @rockyrhodes: It has been 106 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by davidparks21** [comment]
I'm also encountering these errors in trying to build a demo model to run to the hexagon DSP chip. In TF Light I see good work on quantization (though a little sparse in terms of references to exactly what technique is being applied there), but I don't see any reference in TF Light to running those quantized models on the Hexagon DSP. Has there been more recent work in TF Light to support platforms like the Hexagon DSP or mobile GPUs? Is `contrib/hvx` still the right place place to look to work with the Hexagon DSP?

https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/hvx

**Comment by rockyrhodes** [comment]
@satok16 is the only one I know that has really worked on this, and it doesn't appear that he has been involved in it for some time.  TFLite's NNAPI delegate supports the Hexagon DSP and mobile GPU's indirectly through the vendor's NNAPI HAL drivers.  Have you tried this acceleration path?

**Comment by davidparks21** [comment]
I see, thanks for the reply. I'll try to set up a demo using Tensorflow Lite and the android NNAPI and see how it goes. I'm currious where XLA is going then? I was under the impression that this hvx contrib effort was a start at generic accelerated device support via XLA/LLVM.

**Comment by petewarden** [comment]
Closing this, since the direct HVX support in TensorFlow is deprecated in favor of the NN API through TF Lite.
